specs/driver_spec.rb

```diff
@@ -55,13 +63,27 @@
     end
   end
 
-  describe "average_rating method" do
-    before do
+  describe "Driver Status" do
+    it 'changes status if end_time equals nil' do
+      start_time = Time.parse('2015-05-20T12:14:00+00:00')
+      end_time = nil
```

Comment:
You should also check that it _doesn't_ change if you add a trip where the end time is non-nil.